Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "qr_code"
 description = "QR code encoder in Rust, support structured append (data in multiple qrcodes)"
 license = "MIT OR Apache-2.0"
-version = "1.1.0"
+version = "1.2.0"
```

Comment:
Should this bump up the version to 2.0 to indicate that there are incompatible API changes?   (Because of the removal of `pub struct **Optimizer<I>` in https://github.com/RCasatta/qr_code/commit/4c3b949e58b53f1230c146c46743cdebde2adf54)

Reply:
Yes, sorry